app/controllers/new-signup.js

```diff
@@ -91,14 +123,35 @@ export default class NewSignUpController extends Controller {
           this.error = ERROR_MESSAGES.others;
           this.isButtonDisabled = false;
         }
-      })
-      .catch(() => {
+      } catch (error) {
         this.analytics.trackEvent(NEW_SIGNUP_FLOW.UNABLE_TO_REGISTER);
         this.error = ERROR_MESSAGES.others;
         this.isButtonDisabled = false;
-      })
-      .finally(() => {
+      } finally {
         this.isLoading = false;
-      });
+      }
+    } else {
+      //this will get removed after removing feature flag
+      registerUser(signupDetails)
+        .then((res) => {
+          if (res.status === 204) {
```

Comment:
Why do we have status code 204 here?

Reply:
after data updated in backend then the backend send response 204